### PR TITLE
Add CRs with CP Docker images for each release

### PR DESCRIPTION
**Pull request by rohit2b**
This PR provides a CR yaml example for each CP version release - 6.0, 6.1 and 6.2.

### CODE REVIEW

File: general/cp-version/confluent-platform-6.2.0.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+# Confluent Replicator is built as a connector, and so will use the `Connect` CRD.
+kind: Connect
+metadata:
+  name: replicator
+  namespace: confluent
+spec:
+  replicas: 1
+  image:
+    # Use the `cp-enterprise-replicator-operator` Docker image, that contains the Replicator jars
+    application: confluentinc/cp-enterprise-replicator-operator:6.1.1.0
```
[the remark this answers is not in the document]
Comment (rohit2b) [reply general/cp-version/confluent-platform-6.2.0.yaml:47]: @xnulinu what images should we use here for CP 6.2.0?

File: general/cp-version/confluent-platform-6.0.0.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+metadata:
+  name: kafka
+  namespace: confluent
+spec:
+  replicas: 3
+  image:
+    application: confluentinc/cp-server-operator:6.0.0.0
+    init: confluentinc/cp-init-container-operator:6.0.0.0
+  dataVolumeCapacity: 10Gi
+  metricReporter:
```
Comment (xnulinu) [inline general/cp-version/confluent-platform-6.0.0.yaml:26]: nit: not required, enabled automatically

**Review by xnulinu — CHANGES_REQUESTED** [review]

File: general/cp-version/confluent-platform-6.0.0.yaml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    enabled: true
```
Comment (rohit2b) [inline general/cp-version/confluent-platform-6.0.0.yaml:27]:
```suggestion
```

**Review by xnulinu — APPROVED** [review]
+1 Thanks

**Pull request merged**